.github/workflows/windows-build-and-test.yaml

```diff
@@ -177,43 +184,63 @@ jobs:
         # isolationtester is only packaged with pg14+ so we would have to build our own postgres
         # to get it for earlier versions so we skip it for < 14.
         if [[ "${{ matrix.pg }}" == "14" ]]; then
-          make -C build_wsl isolationchecklocal-t
+          make -C build_wsl isolationchecklocal-t | tee -a installcheck.log
         fi
 
-        make -C build_wsl -k regresschecklocal-t IGNORES="${{ matrix.tsl_ignores }}" SKIPS="${{ matrix.tsl_skips }} ${{ matrix.tsl_skips_version }}"
+        make -C build_wsl -k regresschecklocal-t IGNORES="${{ matrix.tsl_ignores }}" SKIPS="${{ matrix.tsl_skips }} ${{ matrix.tsl_skips_version }}" | tee -a installcheck.log
 
     - name: Show regression diffs
-      if: always() && matrix.pg != '12'
-      shell: python
       id: collectlogs
+      if: always() && matrix.pg != '12'
+      env:
+        WSLENV: GITHUB_OUTPUT
+      shell: wsl-bash {0}
       run: |
-        import re
-        import os
-        from pathlib import Path
-
-        for path in Path('build_wsl').rglob('regression.out'):
-          for line in path.open():
-            if re.search('failed', line, re.IGNORECASE):
-              print(line, end='')
-
-        for path in Path('build_wsl').rglob('regression.diffs'):
-          for line in path.open():
-            print(line, end='')
-        
-        with open(os.environ['GITHUB_OUTPUT'], 'a') as output:
-          print('regression_diff=true', file=output)
+        find . -name regression.diffs -exec cat {} + > regression.log
```

Comment:
Why remove the python code here? for convenience or is there another reason?

Reply:
I wanted to make it as similar as possible to the other checks like Linux. This way it's easier to maintain, I can just diff those files and copy the fragments of the code, no need to write same thing anew in another language.